mingw-w64-git/PKGBUILD

```diff
@@ -67,6 +69,9 @@ build() {
 	compat/win32/git-wrapper.o git.res -lshlwapi &&
   gcc -mwindows -s -O2 -Wall -o cmd/gitk.exe gitk.res \
 	compat/win32/git-wrapper.o git.res -lshlwapi
+
+  # git-credential-wincred.exe
+  CC=gcc make -C contrib/credential/wincred
```

Comment:
Yep, `contrib/credential/wincred/Makefile` does not include our `config.mak.uname` which [sets `CC = gcc` explicitly](https://github.com/git-for-windows/git/blob/e992745d6119cae6421abf9326b63c5f9dcc4567/config.mak.uname#L549).